### PR TITLE
#106 | readiness metric and handler interface

### PR DESCRIPTION
**Pull request by skyrrt**

### CODE REVIEW

File: envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/infrastructure/consul/ConsulWatcherConfig.kt [hunk 1]
```diff
@@ -108,5 +108,5 @@ open class ConsulWatcherConfig {
     }
 
     @Bean
-    fun kotlinModule() = KotlinModule()
+    fun kotlinModule() = KotlinModule.Builder().build()
```
Comment (Ferdudas97) [inline envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/infrastructure/consul/ConsulWatcherConfig.kt:111]: why this change?

**Review by Ferdudas97 — APPROVED** [review]

File: envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/infrastructure/consul/ConsulWatcherConfig.kt (hunk 1 above)
Comment (skyrrt) [reply envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/infrastructure/consul/ConsulWatcherConfig.kt:111]: after upgrading Andamio default constructor KotrlinModule() stopped working (and it was also deprecated before) so i changed it to Builder. all defaults are the same so the behaviour is the same

**Review by KSmigielski — APPROVED** [review]
